src/subdomains/supporting/address-pool/deposit/deposit.controller.ts

```diff
@@ -18,4 +18,12 @@ export class DepositController {
   async createDeposits(@Body() dto: CreateDepositDto): Promise<void> {
     await this.depositService.createDeposits(dto);
   }
+
+  @Put('updateLightningDepositWebhook')
```

Comment:
Path is not matching our standard endpoint path naming or?
Maybe something like `lightning/webhook` or `lightning/depositWebhook` would make more sense? 🤔